### ISSUE TITLE
[Bug]: Ship missing at first a jump point or planet in automatic landing mode

### ISSUE THREAD

**Issue by BM-W1** [naev/naev#2422]
### What happened?

Sometimes when approaching a jump point or planet in automatic flight mode, the ship breaks too early. It is then accelerates and breaks several times until arriving at the right location. This takes sometimes quite a lot of time, especially, when you are presecuted by pirates or so. ;-)
I'm sorry I cannot tell when this happens exactly. I believe it is not the case in the 0.10.x versions.

### Version

0.11.0 alpha (current)

### What operating system are you seeing the problem on?

Linux

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by bobbens** [comment]
Is this with reverse thrusters only?

**Comment by bobbens** [comment]
Also is there a particular ship you see this issue with?

**Comment by oo13** [comment]
Is the issue a same as this?
![Peek 2023-10-27 20-04](https://github.com/naev/naev/assets/22392249/bac27c4a-7887-4d36-ab34-0537e8848b22)

I suspect that minbrakedist() takes no account of the slowdown during turning.

How to reproduce it:
1. Take off Antlejos V.
2. Take enough distance to Antlejos V.
3. Select Antlejos V and push L.

Equipments:
U1: Milspec Orion 3701 Core System
U2: Air Freshener
S1: Tricon Zephyr Engine
S2: Unicorp D-2 Light Plating
S3: Improved Stabilizer
S4: Improved Stabilizer
I1: Port & Polish
I2: Machiavellian Fortune

**Comment by bobbens** [comment]
> I suspect that minbrakedist() takes no account of the slowdown during turning.

That's a nice point. I've corrected it somewhat (still not exact though...), to take into account the true max velocity while turning. Although I wasn't able to reproduce, when lowering to 30 FPS, I was able to see a better behaviour with the changes. So hopefully it's fixed. If necessary, I can try to figure out the exact math and get a more exact estimate.

Side note, the Antlejos system was great for testing the lanes autonav, and found a few issues that I was able to solve. Thanks to the both of you!

I'll keep this open until someone confirms it's better.
